ruby/engine/InitRubyEngine.cpp

```diff
@@ -22,6 +25,8 @@ void RubyEngine::initRubyEngine() {
   RUBY_INIT_STACK;
   ruby_init();
 
+  Init_ruby_description(nullptr);
```

Comment:
Use it.

We don't just MJIT or YJIT, so it just doesn't use the param `ruby_cmdline_options_t *opt` anyways.

See https://github.com/ruby/ruby/blob/e51014f9c05aa65cbf203442d37fef7c12390015/version.c#L129-L162